alerts.yaml

```diff
@@ -96,3 +96,12 @@ serverFiles:
           annotations:
             summary: 'Kube Bench failed some checks'
             description: 'The most recent Kube Bench run detected some failures'
+      - name: consumer_group_lag
+        rules:
+        - alert: ConsumerGroupLagLarger
+          expr: sum(kafka_consumergroup_group_lag) > 5000
```

Comment:
I believe we want to know what specifically is lagging, rather than all lag across the entire system.